### DISCUSSION TITLE
does not work on ios mobile devices

### DISCUSSION THREAD

**Discussion by KakNaZlo** [google/model-viewer#3870] [Q&A]
Hi
The 3d model display functionality on ios devices is no longer working
https://raw.githubusercontent.com/KakNaZlo/yagodka/master/files/turbofit_mobile_1/img/image_2022-09-19_13-10-56.png

**Comment by elalish** [comment]
Is this due to a MV version change? Can you link to a working and non-working version so we can debug if so? Generally this error occurs due to an iOS update where Apple has changed their USDZ definition and any existing `ios-src` has to be updated with a USDZ re-exported with the latest version of Reality Converter. @grorg may be able to comment from Apple.
